<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1529</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Afficher les machines</string>
   </property>
   <widget class="QTableView" name="tableView_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>491</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_176">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Extraire</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>70</y>
     <width>291</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_35">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Rechercher les machines</string>
   </property>
   <widget class="QPushButton" name="pushButton_177">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par Libellé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_178">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par Prix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_179">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Par Etat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_h_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_j_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>460</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_57">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_180">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Afficher Historique</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_56">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Trier les machines</string>
   </property>
   <widget class="QPushButton" name="pushButton_181">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>PrixDecroissant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_182">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>PrixCroissant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_183">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string> Libellé</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Modifier ou Supprimer une machine</string>
   </property>
   <widget class="QPushButton" name="pushButton_184">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_185">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_k_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="qte_M_51">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;id_machine&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>421</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Ajouter une machine</string>
   </property>
   <widget class="QPushButton" name="pushButton_VAL_AJ_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QUIT_AJ_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_a_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_machine_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Id_machine</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Libelle_M&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Prix_M&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="fournisseur_M_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Fournisseur_M&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="etat_M_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Etat_M&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_b_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_c_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_d_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_e_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="qte_M_52">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Qte_M&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_f_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="qte_M_53">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Id_EMP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_g_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>0</y>
     <width>1571</width>
     <height>751</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/15190.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>0</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/Artboard-35-ConvertImage.png</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>120</y>
     <width>251</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>ARDUINO</string>
   </property>
   <widget class="QPushButton" name="pushButton_refuser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>REFUSER ACCES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
gridline-color: rgb(0, 0, 0);
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acces">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color: rgb(0, 74, 108);</string>
    </property>
    <property name="text">
     <string>DONNER ACCES</string>
    </property>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>textEdit</zorder>
  <zorder>groupBox_35</zorder>
  <zorder>textBrowser</zorder>
  <zorder>groupBox_57</zorder>
  <zorder>groupBox_56</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_56</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <resources>
  <include location="recource.qrc"/>
 </resources>
 <connections/>
</ui>
